package.json

```diff
@@ -7,7 +7,7 @@
     "install": "for d in packages/*/ ; do (cd \"$d\" && npm install); done",
     "lint": "eslint --config ./eslint/eslintrc packages",
     "flow": "flow",
-    "gen-snapshot-tests": "node utils/gen-snapshot-tests.js",
+    "gen-snapshot-tests": "node utils/gen-snapshot-tests.js && prettier --write packages/*/generated-snapshot.test.js",
```

Comment:
@kevinbarabash I added this in and now files are being properly formatted all the time!